markdown/manual.md

```diff
@@ -1,220 +1,245 @@
+    - Patronizing or belittling a person or group,
+    - Repeatedly interrupting or talking over someone else,
+    - Pedantic corrections that don’t add value,
+    - Expressing an assumption that persons or groups need concepts defined or explained to them without first asking if they do,
+    - Using unsubstantiated language that creates or furthers a belief which diminishes the value or contributions of a person or group,
+    - Employing racially or ethnically charged language to describe a person or thing,
+    - Referring to an individual in a way that demeans or challenges the validity of any aspect of their personal identity, or
+    - Mocking someone’s real or perceived accent or first language.
 
-The company will investigate all such reports as confidentially as possible. Adverse action will not be taken against an employee because he or she, in good faith, reports or participates in the investigation of a violation of this policy. Violations of this policy are not permitted and may result in disciplinary action, up to and including discharge.
+### Reporting and Resolution of Violations
+Any employee who believes they have been harassed or believes they have witnessed another member of our organization violate the terms of this document should report the situation immediately to one of the following members of management: Executive Director, Treasurer or Chairman of the Board of Directors. If an employee makes a report to one of these persons and the employee does not receive a satisfactory response, the employee shall report the situation to one of the other designated persons to receive complaints.
 
+The organization will investigate all such reports as confidentially as possible. Adverse action will not be taken against an employee because he or she, in good faith, reports or participates in the investigation of a violation of this policy. Violations of this policy are not permitted and may result in disciplinary action, up to and including discharge.
 
-### Drug & Alcohol-Free Workplace Statement
-The unlawful or improper presence or use of controlled substances or alcohol in the workplace presents a danger to all employees.  Employees are prohibited from reporting to work or working while using illegal or unauthorized substances.
+### Drug & Alcohol Policy
+The unlawful or improper presence or use of controlled substances or alcohol in the workplace presents a danger to all employees.  All members of the organization are prohibited from reporting to work or working while using illegal or unauthorized substances.
+
+Our organization recognizes that drug or alcohol abuse are serious medical conditions and should be treated as such. Therefore, we shall provide all appropriate organizational support and assistance to those within the organization who request it.  We will also seek out and retain a employer-sponsored health insurance plan that provide drug and alcohol treatment services.
```

Comment:
@batemapf good and stronger.  Thank you.  Our health care coverage covers this already, too, so can change from "We will..." to "Our employer-sponsored..."